pkg/core/mempool/mem_pool.go

```diff
@@ -314,6 +334,14 @@ func (mp *Pool) RemoveStale(isOK func(*transaction.Transaction) bool, feer Feer)
 			}
 		} else {
 			delete(mp.verifiedMap, itm.txn.Hash())
+			if attrs := itm.txn.GetAttributes(transaction.OracleResponseT); len(attrs) != 0 {
+				id := attrs[0].Value.(*transaction.OracleResponse).ID
+				h := mp.oracleResp[id]
+				delete(mp.oracleResp, attrs[0].Value.(*transaction.OracleResponse).ID)
+				if h != itm.txn.Hash() {
```

Comment:
How could it happen if `mp.oracleResp` only contains one transaction for one ID and `itm.txn` (from the pool) already has that ID?

Reply:
When we have 2 transactions with oracle response, one of which was in our mempool and second was in accepted block.

Reply:
Actually no, this can't happen, fixed.